docs/userguides/accounts.md

```diff
@@ -159,6 +174,24 @@ from ape import accounts
 account = accounts.load("<ALIAS>")
 ```
 
+### Default Sender Support
+
+In order to eliminate the usage of sender in contract calls, you can use `use_sender` context manager.
+
+```python
+with accounts.use_sender(0, autosign=True, passphrase="pass"):
+  contract.myFunction(1)
+
+with accounts.use_sender("address", autosign=True, passphrase="pass"):
+  contract.myFunction(1)
+
+with accounts.use_sender("alias", autosign=True, passphrase="pass"):
+  contract.myFunction(1)
+
+with accounts.use_sender(KeyfileAccount, autosign=True, passphrase="pass"):
+  contract.myFunction(1)
```

Comment:
The reason I don't like `passphrase=` kwarg here is that it encourages people to just write their passphrase into a script file they very likely will upload to a repo

If they also upload their keyfile somehow, the funds are as good as gone

I like `unlock=True` better because it just uses the native unlock facility of `KeyfileAccount`

Reply:
Also, how would this work for other account types? (like ledger/trezor)

Reply:
they dont even have unlock. i just looked at TrezorAccount and LedgerAccount implementations, so this is a problem like i wrote in discord

Reply:
i also dont think it would be good idea to eliminate passphrase here and just allow autosign without it, then if you get hacked then it is not fun :D

Reply:
> i also dont think it would be good idea to eliminate passphrase here and just allow autosign without it, then if you get hacked then it is not fun :D

Due to how `KeyfileAccount` works, it can't sign anything without unlocking first because you need the passphrase to decrypt the key